packages/ws-server/src/service/poller.ts

```diff
@@ -28,8 +27,8 @@ export class Poller {
 
   private NEW_HEADS_EVENT = 'newHeads';
 
-  constructor(eth: Eth, logger: Logger, register: Registry) {
-    this.eth = eth;
+  constructor(relay: Relay, logger: Logger, register: Registry) {
```

Comment:
can we keep it `eth: Eth`? instead of depending on the `Relay`?

Reply:
Not sure why. Using Relay adds yet more dependency that we don't use. And it might be also confusing seeing a dependency into the Relay where only `eth` is used.

Given said that, as you prefer. Going to resolve this.